test/paper-menu-button.html

```diff
@@ -28,21 +28,21 @@
   <test-fixture id="TrivialMenuButton">
     <template>
       <paper-menu-button no-animations>
-        <span class="dropdown-trigger">trigger</span>
-        <span class="dropdown-content">content</span>
+        <span slot="dropdown-trigger">trigger</span>
+        <span slot="dropdown-content" class="dropdown-content">content</span>
```

Comment:
Although not strictly necessary, these might be clearer without `class="dropdown-content"`.